pkg/retryablehttp/client.go

```diff
@@ -0,0 +1,107 @@
+package retryablehttp
```

Comment:
The ideal name for this package is probably `http` IMO. I know the conflict with `net/http` could be annoying, but we can alias our package as `wohttp` or something. We should also update the main struct from `HttpClient` to `HttpClientWithRetry` in this case.

Reply:
After discussing, we decided to keep this package named `retryablehttp` and keep the struct named `HttpClient`